documentation/modules/mtv-performance-addendum.adoc

```diff
@@ -0,0 +1,157 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: CONCEPT
+[id="mtv-performance-addendum_{context}"]
+= {project-short} performance addendum
+
+[id="mtv-single-esxi-performance_{context}"]
+== Single ESXi performance
+
+Test migration using the same ESXi host.
+
+In each iteration, the total VMs are increased, to display the impact of concurrent migration on the duration.
+
+The results show that migration time is linear when increasing the total VMs (Utilization 70%).
+
+The optimal number of VMs per ESXi is 10.
```

Comment:
Should this be moved into the single ESXi section?

Reply:
We write the same in the recommendation of Single ESX concurency